--- Makefile.in.orig	2010-12-04 02:05:01.000000000 +0100
+++ Makefile.in	2011-01-05 23:13:32.000000000 +0100
@@ -42,7 +42,6 @@
 endif
 
 # Lists of files.
-BINS := @srcroot@bin/pprof
 CHDRS := @objroot@include/jemalloc/jemalloc@install_suffix@.h \
 	@objroot@include/jemalloc/jemalloc_defs@install_suffix@.h
 CSRCS := @srcroot@src/jemalloc.c @srcroot@src/arena.c @srcroot@src/base.c \
@@ -123,13 +122,6 @@
 	$(CC) -o $@ $< -L@objroot@lib -ljemalloc@install_suffix@
 endif
 
-install_bin:
-	install -d $(BINDIR)
-	@for b in $(BINS); do \
-	echo "install -m 755 $$b $(BINDIR)"; \
-	install -m 755 $$b $(BINDIR); \
-done
-
 install_include:
 	install -d $(INCLUDEDIR)/jemalloc
 	@for h in $(CHDRS); do \
